11/galaxy-requirements.yaml

```diff
@@ -237,9 +237,6 @@ collections:
 - name: ngine_io.cloudstack
   source: https://galaxy.ansible.com
   version: 2.4.1
-- name: ngine_io.exoscale
-  source: https://galaxy.ansible.com
-  version: 1.1.0
```

Comment:
You can leave it in this file, since this is for 10.0.0a1 until 10.0.0a2 is released.